share/src/main/resources/share-config.properties

```diff
@@ -1,8 +1,8 @@
 #Properties for AIMS 
-aims.config.enabled=false
+aims.config.enabled=true
```

Comment:
@alexbalmus Can you keep the aims.config.enabled false, and remove the authServerUrl.